hardware/tv-hat/README.md

```diff
@@ -0,0 +1,58 @@
+
+* Type the following into the address bar: `http://raspberrypi.local:9981/extjs.html`
+* This should connect to TVHeadend running on the Raspberry Pi.
+  * If the address above doesn't work, you’ll need to find out the IP address of the Pi. Open a terminal window on your Pi, and run the command hostname -I
+  * You’ll see the IP address in two formats: a string of four numbers separated by dots, then a space, then a long string of numbers and letters separated by colons.
+  * Copy everything before the space (the four numbers and dots), and paste this into the address bar instead of the raspberrypi.local part of the address.
+* Once you have connected to TVHeadend via the browser, you will be prompted to sign in. Use the account name and password you chose when you installed TVHeadend on the Pi. A setup wizard should appear.
+* First, set the language you want TVHeadend to use (**English (GB)** worked for us; we have
+not yet tested other languages).
+* Next, set up network, user, and administrator access. If you don’t have specific preferences, leave **Allowed network** blank, and enter an asterisk (*) in the **username** and **password** fields. This will let anyone connected to your local network access TVHeadend.
+* You should see a window titled **Network settings**. Under **Network 2**, you should see `Tuner: Sony CDX2880 #0 : DVB-T #0`. For **Network type**, choose `DVB-T Network`.
+* The next window is **Assign predefined muxes to networks**; here, you select the TV stream to receive and decode. Under Network 1, for Pre-defined muxes, select your local TV transmitter.
+  * A list of UK transmitters can be found at [www.digitaluk.co.uk/coveragechecker](www.digitaluk.co.uk/coveragechecker). Enter your postcode to see which transmitter should give you a good signal.
+* When you click **Save & Next**, the software will start scanning for the selected mux, and will show a progress bar. After about two minutes, you should see something like:
+```
+Found muxes: 8
+Found services: 172
+```
+* In the next window, titled **Service mapping**, tick all three boxes: `Map all services`, `Create provider tags`, and `Create network tags`.
+* Next you should see a list of TV channels you can watch, along with the programmes they’re currently showing.
+* To watch a TV channel in the browser, click the little TV icon to the left of the channel
+listing, just to the right of the `i` icon. This brings up an in-browser media player. Depending on the decoding facilities  built into your browser and the type of stream being played, you may find that playback can be jerky. In these cases, we recommend using a local media player as the playback application.
+* To watch a TV channel in a local media player, e.g. VLC [www.videolan.org/vlc](www.videolan.org/vlc), you’ll need to download it: click the `i` icon to the left of a channel listing to bring up the information panel for that channel. Here you can see a stream file that you can download.
```

Comment:
Again just being pedantic - it's not clear whether "download it" refers to downloading VLC, or downloading the TV stream?